superset/connectors/sqla/models.py

```diff
@@ -1368,10 +1368,17 @@ def _get_top_groups(  # pylint: disable=no-self-use
 
         return or_(*groups)
 
+    def _format_sql_str(self, sql: str) -> String:
```

Comment:
let's add a docblock and unit test for this new function? it should be pretty trivial.

Also, what if the sql is something like:
```sql
select * from foo;



```

(note the new lines)

will this still work? Maybe it's worth adding a trim to the format function as well, and adding test cases for it too.

Reply:
Yes the new lines will still work with this. Will be adding unit test!